pkg/cloudprovider/providers/azure/azure.go

```diff
@@ -59,6 +59,9 @@ const (
 	vmTypeVMSS     = "vmss"
 	vmTypeStandard = "standard"
 
+	backoffModeDefault = "default"
+	backoffModeMinimum = "minimum"
```

Comment:
maybe `v2` is better.

Reply:
@brendandburns @khenidak What do you think about the new backoff mode name? I don't think "minimum" is good enough. What's your suggestion?